src/status_im/contexts/wallet/send/events.cljs

```diff
@@ -768,3 +768,41 @@
             :always             (update-in [:wallet :ui :send] dissoc :amount :route)
             (not keep-tx-data?) (update-in [:wallet :ui :send] dissoc :tx-type))
       :fx [[:dispatch [:navigate-back]]]})))
+
+(rf/reg-event-fx
```

Comment:
I do not know yet in what form we will get the recommended and current values for transaction settings from `status-go`, so for now I've stored test values like that. Not a final form I believe.